files/image_config/copp/copp_cfg.j2

```diff
@@ -60,7 +60,10 @@
 		    "trap_action": "trap",
 		    "trap_priority": "1"
 
-	    }
+	    }{%- if include_p4rt == "y" %},
+	    "trap.group.send_to_ingress" : {
+		    "submit_to_ingress_name" : "send_to_ingress"
```

Comment:
Can we move this to the CONFIG_DB port table?
Issue: we can't specify that this is a host interface for CPU there today.

Can we move this to the P4Orch constructor?

Reply:
CoPP group has policers and other things. It doesn't really fit here.

Reply:
Proposal for MVP: put it in P4Orch
Longer term: make this part of the PORT_TABLE